app/poros/api_obituary.rb

```diff
@@ -23,6 +23,6 @@ def format_date(date)
   end
 
   def format_image(url)
-    "https://static01.nyt.com/".concat(url)
+    url ? "https://www.nytimes.com/".concat(url) : 'https://d1yjjnpx0p53s8.cloudfront.net/styles/logo-thumbnail/s3/0013/5263/brand.gif?itok=MkqjUMfP'
```

Comment:
This adds a default image if the NYTimes doesn't return one. I just chose a random image and it isn't formatted the same as the NYTimes thumbnails so we will need to style this, but I needed it to stop the page from throwing an error.

Reply:
Good problem solving.

Reply:
THANK YOU YAAAAYYYYYY